eatin-client/src/components/hooks/useGetRecipesComments.ts

```diff
@@ -0,0 +1,73 @@
+import { useCallback, useEffect, useState } from "react";
+import {
+    useGetCommentsByRecipeIndexQuery,
+    useCreateCommentsMutation,
+} from "../../generated/graphql";
+import { useGetUsersName } from "./useGetUsersName";
+import { v4 as uuidv4 } from "uuid";
+import { Comments } from "../types";
+
+export function useGetRecipesComments(recipeIndex: number) {
+    const userID = useGetUsersName();
```

Comment:
If you get the userId, why is it called getUsersName?

Reply:
Welp dunno

Reply:
so maybe change it...?